server/plugin.go

```diff
@@ -119,6 +113,12 @@ type Plugin struct {
 	otsStore      OTSStore
 	secretsStore  SecretsStore
 
+	setupFlow *flow.Flow
+
+	// Most of ServeHTTP does not use an http router, but we need one to support
+	// the setup wizard flow. Introducing it here incrementally.
+	gorillaRouter *mux.Router
```

Comment:
Lev, can you detail a bit why `serveHTTP` is not enough and we need the `gorillaRouter`? Thanks.